server/reddit.js

```diff
@@ -1,17 +1,31 @@
+/**
+ * Access Reddit user and post data
+ * @exports getContest @returns {entries, isContestMode}
+ * @exports getUser
+ * @exports getWinners
+ * @exports isModerator
+ * @exports retrieveAccessToken
+ * @exports revokeRefreshToken
+ */
+
 const axios = require('axios');
 const Snoowrap = require('snoowrap');
 
+const { IS_DEV } = require('./env');
 const { createLogger } = require('./logger');
 
 const logger = createLogger('REDDIT');
 
 const {
   REDDIT_CLIENT_ID,
   REDDIT_CLIENT_SECRET,
+  REDDIT_USERNAME,
   REDDIT_PASSWORD,
   WEB_APP_CLIENT_ID,
   WEB_APP_CLIENT_SECRET,
   WEB_APP_REDIRECT_URI,
+  WEB_APP_REFRESH_TOKEN,
+  WEB_APP_ACCESS_TOKEN,
```

Comment:
Nit: Could you sort this list alphabetically?

Reply:
Done